docs/excel/excel-add-ins-ranges-set-get.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Set and get ranges using the Excel JavaScript API
```

Comment:
This article is about getting and setting the selected range, so the title (and other bits) so reflect that.
```suggestion
# Set and get the selected range using the Excel JavaScript API
```